Code/src/megadesk.cpp

```diff
@@ -200,66 +202,65 @@ void setup()
       }
       delay(SHORT_PAUSE);
     }
+
   }
 
 #ifdef SERIALCOMMS
   Serial1.begin(19200);
 #endif
 
   // init + arpeggio
-  beep(NOTE_C7);
+  beep(NOTE_C6);
+
   initAndReadEEPROM(false);
-  beep(NOTE_E7);
+  beep(NOTE_E6);
+
   lin.begin(19200);
-  beep(NOTE_G7);
+  beep(NOTE_G6);
 
   linInit();
-  beep(NOTE_C8);
+  // hold up button on boot to toggle both/single Button Mode
+  if (!digitalRead(PIN_UP))
+  {
+    toggleBothMode();
+  } else
+    beep(NOTE_C7, bothmode+1);
```

Comment:
moved this to the end of setup() so that eeprom is prepared already and we can provide feedback with the final note of the arpeggio.
You do have to hold the up-button until the startup sound completes.